src/detectors/ZDC/ZDC.cc

```diff
@@ -102,5 +106,150 @@ extern "C" {
             app   // TODO: Remove me once fixed
           )
         );
+
+        //side length of hexagonal cells in SiPM-on-tile part of the ZDC
+        auto side_length = 31.3 * dd4hep::mm;
+
+        app->Add(new JOmniFactoryGeneratorT<CalorimeterHitDigi_factory>(
+          "ZDCRawHits", {"HcalFarForwardZDCHits"}, {"ZDCRawHits"},
+          {
+            .tRes = 0.0 * dd4hep::ns,
+            .capADC = 32768,
+            .dyRangeADC = 800 * dd4hep::MeV,
+            .pedMeanADC = 400,
+            .pedSigmaADC = 10,
+            .resolutionTDC = 10 * dd4hep::picosecond,
+            .corrMeanScale = 1.0,
+          },
+          app   // TODO: Remove me once fixed
+        ));
+
+        app->Add(new JOmniFactoryGeneratorT<CalorimeterHitReco_factory>(
+          "ZDCRecHits", {"ZDCRawHits"}, {"ZDCRecHits"},
+          {
+            .capADC = 32678,
+            .dyRangeADC = 800. * dd4hep::MeV,
+            .pedMeanADC = 400,
+            .pedSigmaADC = 10,
+            .resolutionTDC = 10 * dd4hep::picosecond,
+            .thresholdFactor = 0.0,
+            .thresholdValue = -100.0,
+            .sampFrac = 1.0,
+            .readout = "HcalFarForwardZDCHits",
+            .layerField = "layer",
+            .sectorField = "system",
+          },
+          app   // TODO: Remove me once fixed
+        ));
+
+        app->Add(new JOmniFactoryGeneratorT<HEXPLIT_factory>(
+          "ZDCSubcellHits", {"ZDCRecHits"}, {"ZDCSubcellHits"},
+          {
+            .layer_spacing=25.1*dd4hep::mm,
+            .side_length = side_length,
+            .MIP = 472. * dd4hep::keV,
+            .Emin_in_MIPs=0.1,
+            .tmax=320 * dd4hep::ns,
+            .trans_x=0,
+            .trans_y=0,
+            .trans_z=36601 * dd4hep::mm,
+            .rot_x=0,
+            .rot_y=-0.025,
```

Comment:
Crossing angle should not be in reconstruction configuration.

Reply:
fixed in commit 22dfc61